### PR TITLE
add Contrib_X Docs group

### PR DESCRIPTION
**Pull request by durandom**
cc @sallyom @stefwrite @quaid 

am I missing somebody else?

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -75,6 +75,29 @@ sigs:
...
+    - github: stefwrite
+      name: Stephanie E. Watson
+      company: Red Hat
+  meetings:
+  - description: Regular WG Meeting
+    day: Wednesday
+    time: "10:00"
+    tz: ET (East Time)
+    frequency: weekly
+    url: https://meet.google.com/...
```
Comment (durandom) [inline sigs.yaml:99]: @quaid can you find a timeslot and a meeting URL?

**Comment by durandom** [comment]
more can be found at  https://github.com/operate-first/community/issues/131

**Review by schwesig — APPROVED** [review]
/lgtm
/approve

decided together in the Community meeting

**Pull request merged**